packages/eslint-plugin-pf-codemods/lib/rules/v5/menu-moved-ariaLabel.js

```diff
@@ -0,0 +1,64 @@
+const { getPackageImports } = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8649
+module.exports = {
+  meta: { fixable: "code" },
+  create: function (context) {
+    const menuImports = getPackageImports(
+      context,
+      "@patternfly/react-core"
+    ).filter((specifier) => specifier.imported.name == "Menu");
+    const getAriaLabel = (node) =>
+      node.openingElement.attributes.find(
+        (attr) => attr?.name?.name === "aria-label"
+      );
+
+    return menuImports.length === 0
+      ? {}
+      : {
+          JSXElement(node) {
+            const menuAriaLabel = getAriaLabel(node);
+
+            if (
+              menuImports
+                .map((imp) => imp.local.name)
+                .includes(node.openingElement.name.name) &&
+              menuAriaLabel
+            ) {
+              const menuLists = node.children.filter(
+                (child) => child?.openingElement?.name?.name === "MenuList"
+              );
+
+              context.report({
+                node,
+                message: `The aria-label prop has been removed from Menu and should be passed into MenuList instead. If using MenuGroup with the "label" prop, an aria-label on MenuList is not necessary.`,
+                fix(fixer) {
+                  const fixes = [];
+                  fixes.push(fixer.replaceText(menuAriaLabel, ""));
+
+                  if (
+                    menuLists.length === 1 &&
```

Comment:
we wouldn't want to do anything with multiple lists? i'm not really familiar with how these might be set up

Reply:
My thought was if there are multiple lists within a single Menu, consumers should go through them to apply an accurate aria-label based on the context rather than each list being given the same label.